btk/survey.py

```diff
@@ -1,25 +1,41 @@
 import descwl
 
+surveys = {
+    "LSST": {"bands": ("y", "z", "i", "r", "g", "u"), "pixel_scale": 0.2},
+    "DES": {"bands": ("i", "r", "g", "z"), "pixel_scale": 0.263},
+    "CFHT": {"bands": ("i", "r"), "pixel_scale": 0.185},
+    "HSC": {
+        "bands": (
+            "y",
+            "z",
+            "i",
+            "r",
+            "g",
+        ),
+        "pixel_scale": 0.17,
+    },
+}
+
 
 class Survey(descwl.survey.Survey):
```

Comment:
The name `Survey` might be confusing, since it's actually like a postage stamp because of the inclusion of WCS. Maybe `ObsPostage`, `ObsCutout`, or something is better.

Reply:
Yes I agree that the 'Survey' name might be confusing. What do you think @thuiop, @herjy ?

Reply:
I like `ObsCutout`